src/index.html

```diff
@@ -139,11 +185,7 @@ <h1 class="nav-title">Cape Agulhas</h1>
             </a>
           </div>
         </div>
-        <div class="tab-contents__wrap">
-          <div class="tab-content scroll">
-            <div class="grid--default"></div>
```

Comment:
I think this has broken the JS- an assertion is failing:

`Assertion failed: console.assert` that asserts that the length of `tab-content` is 1